README.md

```diff
@@ -642,21 +642,41 @@ retain("<sourceField_1>"[, ...])
 
 ##### `set_array`
 
-_Currently alias for [`add_array`](#add_array)._
+Creates a new array (with optional values), provided that the intermediate structures (i.e. parent fields) exist.
```

Comment:
We could still refer to the corresponding `add_*` function as an alternative. WDYT?

Reply:
Thanks for the suggestions. Applied in 2c297fa.